### DISCUSSION TITLE
Améliorer les items du shop

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#582] [Nouvelles idées]
Pouvoir choisir le type d'item acheté par exemple ou la rareté, quitte à payer plus cher.

**Comment by Pietagorh** [comment]
Pouvoir miser un de ses items actuels pour avoir plus chance de tomber sur un bon item ?

**Comment by Chevalier2** [comment]
Je ne pense pas que cela soit une bonne idée, en effet si tu as un objet que l'on peut qualifier de nul, tu ne perds " rien ".

**Reply by Pietagorh** [reply]
Bah on fait juste qu'en fonction de la rareté de l'item tu as différentes chances
